<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nanYou::KouLingYanZheng</class>
 <widget class="QWidget" name="nanYou::KouLingYanZheng">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Â§öÂ™í‰ΩìÁÆ°ÁêÜÁ≥ªÁªü</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-smile-big">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #eef2f8;
border-color: rgb(7, 5, 41);
border: 1px;
border-radius: 10px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="avatar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/assets/images/avatar.png);
</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size:30px;
</string>
       </property>
       <property name="text">
        <string>ÁôªÂΩï</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>9</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: gray;</string>
       </property>
       <property name="text">
        <string>ËØ∑ËæìÂÖ•ÂØÜÁ†ÅÁôªÂΩïÁ≥ªÁªü</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="_z_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>134</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">margin-right: 10px;
font-size: 16px;</string>
       </property>
       <property name="text">
        <string>üîë ÂØÜÁ†Å</string>
       </property>
       <property name="buddy">
        <cstring>miMaShuRu</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="miMaShuRu">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 8px;
border-bottom-color: white;
background-color:white;
padding: 8px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="queRenAnNiu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: #429c82;
padding: 10px;
border: 0;
border-radius: 18px;</string>
     </property>
     <property name="text">
      <string>Á°ÆËÆ§ÁôªÂΩï</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
